### ISSUE TITLE
Editor's draft oddity - missing some of the notes in the quoted WCAG success criteria

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#311]
Just noticed today that there are 2 anomalies in the content included from WCAG's GitHub files into WCAG2ICT. The issue is with notes for the SC from WCAG that do not appear or are not the same text in the quoted content vs. what is in the WCAG 2.2 Recommendation. 

- It seems [1.4.12 Text Spacing in WCAG2ICT](https://w3c.github.io/wcag2ict/#text-spacing) is being taken from the WCAG 2.1 part of the GitHub repository. Both of the following notes (introduced in WCAG 2.2) are missing from the quoted text that is contained in the [WCAG 2.2 version of 1.4.12](https://www.w3.org/TR/WCAG22/#text-spacing).
    >NOTE 1: Content is not required to use these text spacing values. The requirement is to ensure that when a user overrides the authored text spacing, content or functionality is not lost.
    >
    >NOTE 2: Writing systems for some languages use different text spacing settings, such as paragraph start indent. Authors are encouraged to follow locally available guidance for improving readability and legibility of text in their writing system. 
- [2.4.11 Focus Not Obscured (Minimum) in WCAG2ICT](https://w3c.github.io/wcag2ict/#focus-not-obscured-minimum) - has a single note, as follows:
    >If the interface is configurable so that the user can reposition content such as toolbars and non-modal dialogs, then only the initial positions of user-movable content are considered for testing and conformance of this Success Criterion.

    Where the [WCAG 2.2 version](https://www.w3.org/TR/WCAG22/#focus-not-obscured-minimum) has 2 notes which are the following:
    >NOTE 1: Where content in a configurable interface can be repositioned by the user, then only the initial positions of user-movable content are considered for testing and conformance of this Success Criterion.
    >
    >NOTE 2: Content opened by the user may obscure the component receiving focus. If the user can reveal the focused component without advancing the keyboard focus, the component with focus is not considered hidden due to author-created content.
 
I thought that when the WCAG2ICT document was built, it was taken directly from the WCAG GitHub repository. These differences seem like they are drawn from a copy (that's older)...or something else odd is going on.

**Comment by maryjom** [comment]
@daniel-montalvo Wondering if you can address this issue I discovered. It seems that something in the script is taking the older, WCAG 2.1 content rather than the newer WCAG 2.2 content for a couple of the SC.
